packs/spell-effects/spell-effect-fiery-body.json

```
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Fiery Body",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You gain fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water.</p>\n<p>Your unarmed attacks deal an additional 1d4 fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses).</p>\n<p>In fire form, you have a fly Speed of 40 feet.</p><hr /><p><strong>Heightened (9th)</strong> Your unarmed attacks deal an additional 2d4 fire damage, and you have a fly Speed of 60 feet.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You gain fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[3d6[fire]] damage.</p>\n<p>Your unarmed attacks deal an additional 1d4 fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). You can cast @UUID[Compendium.pf2e.spells-srd.Item.Ignition] as an innate spell; the casting is reduced from 2 actions to 1.</p>\n<p>In fire form, you have a fly Speed of 40 feet.</p><hr /><p><strong>Heightened (9th)</strong> Your unarmed attacks deal an additional 2d4 fire damage, and you have a fly Speed of 60 feet.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fiery Body]</p>\n<p>You gain fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water.</p>\n<p>Your unarmed attacks deal an additional 1d4 fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses).</p>\n<p>In fire form, you have a fly Speed of 40 feet.</p><hr /><p><strong>Heightened (9th)</strong> Your unarmed attacks deal an additional 2d4 fire damage, and you have a fly Speed of 60 feet.</p>"
```
We don't like to list things effects don't actually automate. The first of these is doable as a note on `damage-received`.